src/utils/slot.py

```diff
@@ -26,9 +27,10 @@ def get_first_non_missed_slot(
     cc: ConsensusClient,
     slot: SlotNumber,
     last_finalized_slot_number: SlotNumber,
+    direction: Literal['back'] | Literal['forward'] = 'back'
```

Comment:
Implementation for the `forward` case is missing

Reply:
The first part of the function is the implementation.